entry/serializers.py

```diff
@@ -76,22 +77,33 @@ class Meta:
 
 
 class EntrySerializer(RemoveNullFieldsMixin,
-                      DynamicFieldsMixin, UserResourceSerializer):
+                      DynamicFieldsMixin, ProjectEntitySerializer):
     attributes = ListToDictField(
         child=SimpleAttributeSerializer(many=True),
         key='widget',
         source='attribute_set',
         required=False,
     )
 
+    project = serializers.PrimaryKeyRelatedField(
+        required=False,
+        queryset=Project.objects.all()
+    )
+
     class Meta:
         model = Entry
-        fields = ('id', 'lead', 'analysis_framework',
+        fields = ('id', 'lead', 'analysis_framework', 'project',
                   'entry_type', 'excerpt', 'image', 'information_date',
                   'attributes', 'order', 'client_id',
                   'created_at', 'created_by', 'modified_at', 'modified_by',
                   'version_id')
 
+    def create(self, data):
+        project = data['project']
```

Comment:
This function seems pointless at the moment. Following should be done instead:

* If `data.get('project')` is None, set `data['project']` to `data['lead'].project`
* Do the same for assessment
* Write test when project is not provided